sdk/eventhub/azure-eventhub/azure/eventhub/_pyamqp/aio/_receiver_async.py

```diff
@@ -31,6 +32,7 @@ def __init__(self, session, handle, source_address, **kwargs):
         self._on_transfer = kwargs.pop("on_transfer")
         self._received_payload = bytearray()
         self._first_frame = None
+        self._received_messages = set()
```

Comment:
nit - we call our received_msg queue this as well don't we, I don't think it matters much but it could be confusing? up to you

Reply:
Ill change it to `received_delivery_tags`